src/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.md

```diff
@@ -44,7 +44,7 @@ You can apply these levels of customization to your site, where the levels requi
 
 ## Frontend development prerequisites {#fedg-prereqs}
 
-To implement what is discussed in this guide, you need a working Magento installation and the following browser versions installed on your device:
+To implement what is discussed in this guide, you need a working Magento installation, and the following browser versions installed on your device:
```

Comment:
As above
```suggestion
To implement what is discussed in this guide, you need a working Magento installation and the following browser versions installed on your device:
```

Reply:
Hi @BarnyShergold 
Thanks for your notes!
Should I revert my changes here?

Reply:
Done!